pkg/trtl/trtl_test.go

```diff
@@ -102,6 +109,15 @@ func generateDB(s *trtlTestSuite) {
 	log.Info().Msg("successfully regenerated test fixtures")
 }
 
+func (s *trtlTestSuite) connect() (*grpc.ClientConn, error) {
+	ctx := context.Background()
+	return grpc.DialContext(ctx, "bufnet", grpc.WithContextDialer(s.dialer), grpc.WithInsecure())
+}
+
+func (s *trtlTestSuite) dialer(context.Context, string) (net.Conn, error) {
+	return s.conn.Dial()
+}
```

Comment:
These functions allow tests to create a client that connects via the bufconn. The general pattern for tests is now implemented in `TestGet` and `TestBatch`.